source/includes/_tipser-react-elements.md

```diff
@@ -196,14 +178,55 @@ Components are the building blocks of Tipser Elements. Any components need to be
 
 `Collection`<br>
 `Product`<br>
-`Store`<br>
+`ProductList`<br>
 `Cart`<br>
+`Store`<br>
 `Checkout`<br>
 
-## Store 
-Store component is a HTML widget displaying a list of all Tipser public collections for the POS.   
+## `Collection`
+Element that renders a collection of product tiles based on [collectionId](#getting-tipser-product-id-and-collection-id) prop. 
+
+If the collection has many elements and you want to display it in one row as a collection, you need to add `carousel` prop. You can also use 'imgSize' prop to control the size of displayed product tiles. [Learn more](#collection-element).
+
+**example: **
+```jsx
+<Collection collectionId={'5b2788909d25801adcb23f4f'} carousel imgSize={'small'} />
+```
+
+## `Product` 
+Element that renders the product title based on Tipser [productId](#getting-tipser-product-id-and-collection-id) passed to the `productId` prop. 
+
+The `Product` element supports two display modes (controlled by `viewMode` prop):
+
+- Large product title with variant selector and "add to cart" button, if `viewMode="full"` (default)
+- Small product title which when clicked opens a product dialog, if `viewMode="compact"`
+
+**example: **
+```jsx
+<Product productId="5c751cf82d3f3b0001bcec8c" viewMode={'compact'} />
+```
+
+[Learn more](#product-element).
+
+## `Product List`
+It is an element that is displayed in the same way as the `Collection`, but instead of the collection Id, you need to pass the array of `productId`s and optional `carousel` and `imgSize`. 
```

Comment:
Może:

> A different variant of `Collection` that accepts an array of `productId's` instead of `collectionId`. 

Pod spodem warto by zrobić tabelkę, w której są wylistowane parametry:
- productIds (required)
- carousel (optional)
- imgSize (optional)

Jeżeli nie chcemy się jeszcze bawić w tabelki, to możemy dalej tylko wspominać o opcjonalnych parametrach w opisie.